docs/source/app-dev/submission-disclosed-contracts.rst

```diff
@@ -0,0 +1,75 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _submission-disclosed-contracts:
+
+(Experimental) Submitting commands with disclosed contracts
+###########################################################
+
+As described in :ref:`Divulgence: When Non-Stakeholders See Contracts <da-model-divulgence>`,
```

Comment:
What I would add here is a start on a positive, a preamble that states that with the version of 2.6 the old conundrum of delegating read rights of a contract **C** to a non-stakeholder party **P** has been solved elegantly by inclusion of the disclosed contract submission. Only then would I proceed to explain what was wrong with the previous two methods.